Jenkinsfile

```diff
@@ -31,3 +31,5 @@ pipeline {
 		}
 	}
 }
+
+// A dummy change to be removed, just to provoke version increments
```

Comment:
Please DO NOT MERGE this PR as it is, as we don't want to have this dummy content end up in this repo!